+sig/+node/Signal.m

```diff
@@ -354,22 +354,48 @@
       l.Node.CurrValue = struct('time', {}, 'value', {});
     end
     
-    function h = onValue(this, fun)
+    function h = onValue(this, fun, noHFlag)
+      % Creates a listener for a signal when the signal takes a value. 
+      % If 'noHFlag' is given a truthy value, the listener can listen
```

Comment:
I would just put 'when "noHFlag" is true'.  'Truthy' is a sort of confusing term that is necessary to distinguish in methods like 'at' but not really here.  

I don't understand what this method is for... Likewise with 'output'

Reply:
ok regarding your first point.

what do you mean by your second point? do you mean because you could instead use 'map' ?

Reply:
Following up on this